.github/workflows/auto-publish.yml

```
@@ -18,12 +18,12 @@ jobs:
        - name: Publish to CurseForge
          uses: Kir-Antipov/mc-publish@v3.3
          with:
-           name: ${{ steps.download-files.outputs.tag_name }} for 1.19.4
+           name: ${{ steps.download-files.outputs.tag_name }} for ${{ vars.MC_VERSION }}
            curseforge-id: 396246
            curseforge-token: ${{ secrets.CURSEFORGE_TOKEN }}
            curseforge-files: '*.zip'
            loaders: fabric
-           game-versions: '1.19.4' 
+           game-versions: ${{ env.MC_VERSION }}
```

Shouldn't this re-use the vars type instead of env? *(saying this as it looks like a duplicate but under different string)*

---

yep, copy/paste moment

---

Does this make sense? I created an environment called "github-actions" and placed _MC_VERSION_ there.
```suggestion
           game-versions: ${{ github-actions.MC_VERSION }}
```

---

Implemented. (docs: https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#using-an-environment)